src/server/actions.ts

```diff
@@ -53,31 +54,33 @@ export default class Actions {
     return this.api;
   }
 
-  async sendTokens (address: string, amount: string): Promise<string | null> {
-    try {
-      const api = await this.getApiInstance();
-
-      if (!this.account) {
-        throw new Error('account not ready');
-      }
+  async sendTokens (address: string, amount: string): Promise<DripResponse> {
+    let dripTimeout: ReturnType<typeof rpcTimeout> | null = null;
+    const result: DripResponse = {};
 
-      logger.info('💸 sending tokens');
+    try {
+      if (!this.account) throw new Error('account not ready');
 
       const dripAmount = Number(amount) * 10 ** decimals;
+      const api = await this.getApiInstance();
       const transfer = api.tx.balances.transfer(address, dripAmount);
 
+      logger.info('💸 sending tokens');
```

Comment:
it makes more sense to log this **before** calling api.tx.balances.transfer

Reply:
We call it when we sign and send.

Reply:
It would be a good idea to divide this log in two messages. If api.tx.balances.transfer throws an exception right now, you never get to this line so it's less clear what happened from the logs.